### ISSUE TITLE
Modal cannot be used if switching to MaterialCss 0.97.8

### ISSUE THREAD

**Issue by gaplo917** [krescruz/angular-materialize#207]
0.97.7 tested, and it is ok. 

According to MaterialCss 0.97.8 Change Log

v0.97.8 (October 30th)
**Refactored Modal plugin**
Tabs now supported in navbar
Chips data can now be reinitiailized
Minor side nav fixes
FAB to toolbar component added
Fixed dropdown options bug

---

Error as shown 0.2.1 version angular-material
```
angular.js:13920 TypeError: element.leanModal is not a function
    at angular-materialize.js:1141
    at angular.js:19612
    at completeOutstandingRequest (angular.js:5964)
    at angular.js:6243
```

**Comment by strongant** [comment]
I also encountered this error, select the drop to 0.97.7

**Comment by tobias-theobald** [comment]
Duplicate of #202.

**Comment by heoheoyh** [comment]
i got same error

**Comment by webbiesdk** [comment]
Closed, duplicate of #202.
